LICENSE

```diff
@@ -321,12 +321,12 @@ For the jwt-go component:
 
 For the pq component:
 @traffic_ops/experimental/traffic_ops_auth/vendor/github.com/lib/pq/*
-@traffic_ops/testing/api/vendor/github.com/lib/pq/*
+@traffic_ops/vendor/github.com/lib/pq/*
 ./traffic_ops/experimental/traffic_ops_auth/vendor/github.com/lib/pq/LICENSE.md
```

Comment:
Looks like this line should be changed to `./traffic_ops/vendor/github.com/lib/pq/LICENSE.md` ?

Reply:
well they all are referencing the license file at the bottom of the list I believe, so the 3 with @'s are all using the LICENSE.md under experimental.